File: algorithm/src/main/java/baekjoon/p06321/Main.java
package baekjoon.p06321;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        for (int i = 1; i <= n; i++) {
            System.out.println(String.format("String #%d", i));
            System.out.println(solve(sc.next()));
            System.out.println();
        }
    }

    private static String solve(String input) {
        StringBuilder sb = new StringBuilder(input.length());
        for (char ch : input.toCharArray()) {
            sb.append((char) ((ch - 'A' + 1) % 26 + 'A'));
        }

        return sb.toString();
    }

}

File: algorithm/src/main/java/baekjoon/p06321/Main.kt
package baekjoon.p06321

fun main(args: Array<String>) {
    val n = readLine()!!.toInt()
    for (i in 1..n) {
        println("String #$i")
        println(solve(readLine()!!))
        println()
    }
}

private fun solve(input: String): String {
    return input.map {
        when (it) {
            'Z' -> 'A'
            else -> it + 1
        }
    }.joinToString("")
}